db/controllers.js

```diff
@@ -0,0 +1,14 @@
+const { selectTopics } = require("./models.js")
+
+exports.getTopics = (req, res) => {
+    console.log("in controller");
+    selectTopics()
+    .then((result) => {
+        console.log(result.rows);
+        res.status(200).send({ topics: result.rows })
```

Comment:
This is fine but I would suggest doing you de structuring of the data within the model rather than result.rows in the controller :) - just something to keep in mind to make it look even neater